test/test_hyperparameters.py

```diff
@@ -578,6 +614,14 @@ def test_ordinal_is_legal(self):
         self.assertFalse(f1.is_legal("chill"))
         self.assertFalse(f1.is_legal(2.5))
         self.assertFalse(f1.is_legal("3"))
+
+        # Test is legal vector
+        self.assertTrue(f1.is_legal_vector(1.0))
+        self.assertTrue(f1.is_legal_vector(0.0))
+        self.assertTrue(f1.is_legal_vector(0))
+        self.assertTrue(f1.is_legal_vector(3))
+        self.assertFalse(f1.is_legal_vector(-0.1))
+        self.assertFalse(f1.is_legal_vector("Hahaha"))
         
     def test_ordinal_check_order(self):
         f1 = OrdinalHyperparameter("temp", 
```

Comment:
Ordinals turned out to be tricky when we added them Could you please add a unit test for them as well?